Makefile

```diff
@@ -77,7 +77,7 @@ prepare-release: cross
 
 .PHONY: test
 test:
-	go test -race $(PKGS)
+	go test -v -race $(PKGS)
```

Comment:
do we really need tests to be verbose?

Reply:
no I have it enabled so that I didn't miss logs for any failures
I'll turn it off once the PR is out of WIP and ready to be in ;)